# multiString message with a header to track times.
Header header

string[] Strings
